js_arrays/task2.js

```diff
@@ -0,0 +1,14 @@
+/* Завдання 2: Копіювання з мутацією
+
+Вам потрібно створити програму, яка мутує значення кожного елемента в масиві за допомогою методу map. 
+ */
+
+const numbers = [1, 2, 3, 4, 5];
+
+function createArray(numbers){
```

Comment:
actually my expectations were a bit different 
```js
const newArray = numbers.map((number, index) => number * index);
console.log(newArray)
```
No need to wrap it in the function